src/middleware/errorHandler.ts

```diff
@@ -1,5 +1,6 @@
-// eslint-disable-next-line
-export default (err, req, res, next) => {
+import { Response } from 'express';
+
+export default (err: any, res: Response) => {
```

Comment:
again, we don't want to use `any` and we don't want to remove all these arguments - please add all these arguments back into this function as it will break the error handler